desktop/core/src/desktop/lib/i18n.py

```diff
@@ -59,8 +60,13 @@ def smart_unicode(s, strings_only=False, errors='strict', encoding=None):
   Wrapper around Django's version, while supplying our configured encoding.
   Decode char array to unicode.
   """
-  return django.utils.encoding.smart_text(
-        s, encoding if encoding is not None else get_site_encoding(), strings_only, errors)
+  if sys.version_info[0] > 2:
+    from django.utils.encoding import smart_str
+  else:
+    from django.utils.encoding import smart_text as smart_str
```

Comment:
Move with other imports? (bad practice to inline)

Reply:
can't do that here because smart_str is a different function in python 2 here I am just using it as a name only. As we are using smart_str import in python 2 here (https://github.com/cloudera/hue/pull/1908/files#diff-e23c3a847d143fa45b3c2c32828ac86e1ff4d3286daa20b0bcff3eb16e9fde11R84)
so there is another option we have like

```
if sys.version_info[0] > 2:
   return django.utils.encoding.**smart_str**(
    s, encoding if encoding is not None else get_site_encoding(), strings_only, errors)
  else:
    return django.utils.encoding.**smart_text**(
    s, encoding if encoding is not None else get_site_encoding(), strings_only, errors)
```
but this code is repeating itself twice.
so what is your call? @romainr

Reply:
Not sure I get it, why not just rename it too if it conflicts? `... as django_smart_str`